### PR TITLE
Chore: esbuild  upgrade and remove form package

### PR DESCRIPTION
**Pull request by SiTaggart**
- Update esbuild from https://github.com/twilio-labs/paste/pull/2166
- Remove form package to resolve some weird 'I can't find textarea' build failure, because it was trying to get it via the form package. We deprecated a long time ago. Time to remove it entirely. Issue resolved.

### CODE REVIEW

File: .changeset/angry-ants-reply.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+'@twilio-paste/nextjs-template': patch
+'@twilio-paste/style-props': patch
+'@twilio-paste/theme': patch
+'@twilio-paste/theme-designer': patch
+'@twilio-paste/token-contrast-checker': patch
+'@twilio-paste/types': patch
+'@twilio-paste/utils': patch
+---
+
+esbuild dependency update for compiled source code
```
Comment (shleewhite) [inline .changeset/angry-ants-reply.md:82]: question: what does this mean?

**Review by shleewhite — APPROVED** [review]

**Comment by shleewhite** [comment]
Looks good, but why are all the packages larger now?

File: .changeset/angry-ants-reply.md (hunk 1 above)
Comment (TheSisb) [reply .changeset/angry-ants-reply.md:82]: suggestion: "Updated a build dependency (esbuild) which changes the output of our builds slightly, without materially changing anything about the code."

File: packages/paste-core/components/form/stories/form.stories.tsx [hunk 2]
```diff
@@ -4,29 +4,38 @@ import {Text} from '@twilio-paste/text';
 import {Box} from '@twilio-paste/box';
 import {Grid, Column} from '@twilio-paste/grid';
 import {Anchor} from '@twilio-paste/anchor';
-import {Checkbox, FormLabel, FormInput, Select, Option, Radio, FormTextArea, CheckboxGroup, RadioGroup} from '../src';
-import type {FormInputProps, SelectProps, FormTextAreaProps, CheckboxGroupProps, RadioGroupProps} from '../src';
+import {Checkbox, CheckboxGroup} from '@twilio-paste/checkbox';
```
Comment (TheSisb) [inline packages/paste-core/components/form/stories/form.stories.tsx:7]: nitpick (non-blocking, out of scope): Keeping the `components/form` folder for a story file can be misleading to future developers of Paste. How would we feel about moving these to the respective package story files?

File: tools/build/esbuild.js [hunk 3]
```diff
@@ -96,6 +97,8 @@ function build(packageJson) {
       minify: true,
       format: 'esm',
       outfile: outFileESM,
+      // Needed to bug with replacing require with import statements https://github.com/evanw/esbuild/issues/566
```
Comment (TheSisb) [inline tools/build/esbuild.js:100]: suggestion: typo in the comment? "Needed to fix a bug..."

**Review by TheSisb — APPROVED** [review]
Finally. And thank you.

**Comment by TheSisb** [comment]
> Looks good, but why are all the packages larger now?

The way esbuild compiles the files has changed so that they need a little more boilerplate. It kinda sucks, but isn't too big a deal after gzipping itll all get compressed away.

**Comment by zahnster** [comment]
Worthy of one of these

eee

File: .changeset/angry-ants-reply.md (hunk 1 above)
Comment (SiTaggart) [reply .changeset/angry-ants-reply.md:82]: Yeah, what he said. Becuase we change the compiled code _ever so slightly_ with this esbuild upgrade, we need to force publish all packages even though nothing actually changes

File: packages/paste-core/components/form/stories/form.stories.tsx (hunk 2 above)
Comment (SiTaggart) [reply packages/paste-core/components/form/stories/form.stories.tsx:7]: I struggled with this. The reason we have this big story is more to capture the spacing between the different type of controls, vertically and horizontally. 

We had a collapsing margin bug once, so we need to ensure the vertical rythmn between all the form controls doesn't degrade. 

But that doesn't give it a good home as it uses all form components... 😬

**Pull request merged**